src/pages/steps/PriceInformation.tsx

```diff
@@ -391,7 +391,7 @@ const PriceInformation = ({
         </Inline>
       ))}
       {hasGlobalError && (
-        <Alert marginTop={3} variant={AlertVariants.INFO}>
+        <Alert display="flex" marginTop={3} variant={AlertVariants.INFO}>
```

Comment:
```suggestion
        <Alert marginTop={3} variant={AlertVariants.INFO}>
```

It could be an idea to just add `display="flex"` in the Alert component itself on the Box component

Reply:
Or use the `Inline` components instead of Box, which is 

```
display: flex;
flex-direction: row;
```

Reply:
Used an inline component ✅ 
Updated screenshots ☝️